src/wordlift/webhooks/api/class-rest-controller.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * @since 3.31.0
+ * @author
+ * Class to call up webhooks and send them requested message
+ */
+
+namespace Wordlift\Webhooks\Api;
+
+use WP_REST_Server;
+
+
+class Rest_Controller {
+
+    /**
+     * Registering the actions to call up sync_many or sync_delete methods
+     */
+
+    public function __construct() {
+	    add_action( 'wl_sync_many', array( $this, 'register_sync_many' ), 10, 2 );
+	    add_action( 'wl_sync_delete', array( $this, 'register_sync_delete' ), 10, 3 );
+    }
+
+    /**
+     * Method to call up webhook with post requested
+     * @param array $payloads
+     * @param array $object
+     * @return json
+     */
+
+	public function register_sync_many( $payloads, $object ) {
+
+        $webhook_urls = get_option('wl_webhook_url');
+        if( ! empty( $webhook_urls ) ) {
```

Comment:
Exit early, that is change this

```
f( ! empty( $webhook_urls ) ) {
```

to 
```
if ( empty( $webhook_urls ) ) {
  return; 
}
```

Reply:
Done